pkgs/tools/backup/discordchatexporter-cli/default.nix

```diff
@@ -16,65 +17,12 @@ stdenv.mkDerivation rec {
     sha256 = "JSYIhd+DNVOKseHtWNNChECR5hKr+ntu1Yyqtnlg8rM=";
   };
 
-  nativeBuildInputs = [ dotnet-sdk_5 dotnetPackages.Nuget cacert makeWrapper autoPatchelfHook ];
-  buildInputs = [ stdenv.cc.cc.lib ];
-
-  nugetDeps = linkFarmFromDrvs "${pname}-nuget-deps" (import ./deps.nix {
-    fetchNuGet = { name, version, sha256 }: fetchurl {
-      name = "nuget-${name}-${version}.nupkg";
-      url = "https://www.nuget.org/api/v2/package/${name}/${version}";
-      inherit sha256;
-    };
-  });
-
-  configurePhase = ''
-    runHook preConfigure
-
-    export HOME=$(mktemp -d)
-    export DOTNET_CLI_TELEMETRY_OPTOUT=1
-    export DOTNET_NOLOGO=1
-
-    nuget sources Add -Name nixos -Source "$PWD/nixos"
-    nuget init "$nugetDeps" "$PWD/nixos"
-
-    # FIXME: https://github.com/NuGet/Home/issues/4413
-    mkdir -p $HOME/.nuget/NuGet
-    cp $HOME/.config/NuGet/NuGet.Config $HOME/.nuget/NuGet
-
-    dotnet restore --source "$PWD/nixos" ${projectFile}
-
-    runHook postConfigure
-  '';
-
-  buildPhase = ''
-    runHook preBuild
-
-    dotnet build ${projectFile} \
-      --no-restore \
-      --configuration Release \
-      -p:Version=${version}
-
-    runHook postBuild
-  '';
-
-  installPhase = ''
-    runHook preInstall
-
-    dotnet publish ${projectFile} \
-      --no-build \
-      --configuration Release \
-      --no-self-contained \
-      --output $out/lib/${pname}
-    shopt -s extglob
-
-    makeWrapper $out/lib/${pname}/DiscordChatExporter.Cli $out/bin/discordchatexporter-cli \
-      --set DOTNET_ROOT "${dotnetCorePackages.sdk_3_1}"
-
-    runHook postInstall
-  '';
+  projectFile = "DiscordChatExporter.Cli/DiscordChatExporter.Cli.csproj";
+  dotnet-runtime = dotnetCorePackages.aspnetcore_3_1;
```

Comment:
Why it's changed from `sdk_5` to `aspnetcore_3_1`?
And can `dotnet-sdk` and `dotnet-runtime` have different version? The former is not overrided here.

Reply:
> Why it's changed from sdk_5 to aspnetcore_3_1?

I guess I didn't pay enough attention when converting this package, it should be `dotnetCorePackages.netcore_3_1`. Changing it to that :)

> And can dotnet-sdk and dotnet-runtime have different version? The former is not overrided here.

Yes, this is actually the only way to use this package, as per the [project's readme](https://github.com/Tyrrrz/DiscordChatExporter#building-the-project-locally).